testing/darktable/APKBUILD

```diff
@@ -37,6 +37,4 @@ package() {
 	make DESTDIR="$pkgdir" install
 }
 
-md5sums="d64520874c0c9606849309b4110a5d73  release-2.2.0rc0.tar.gz"
-sha256sums="67cf6f5a460689b7babb65877c5226ef73155d5134e2e4a2c62ad47156ca394d  release-2.2.0rc0.tar.gz"
-sha512sums="83efe4414420a19239fafdfb5d7ea8bf1bbf3d6cceae97bd8b9e4e65adce4d420dcf0104ed909904eb6fcd4749d6c8bad9f0951de5b63d7cb4d55f0056e260b3  release-2.2.0rc0.tar.gz"
+sha256sums="0612163b0020bc3326909f6d7f7cbd8cfb5cff59b8e0ed1a9e2a2aa17d8f308e  $pkgname-${pkgver/_/.}.tar.xz"
```

Comment:
I did this because they specified the hash on the [release notes][0]. But if you insist, I can change them to the autogenerated ones.

[0]:https://github.com/darktable-org/darktable/releases/tag/release-2.2.0rc1